keyboards/gray_studio/think65v3/config.h

```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 Yizhen Liu (@edwardslau)
+// SPDX-License-Identifier: GPL-2.0
+#pragma once
+#define RGBLIGHT_LAYERS
+#define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF
+#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_MOOD
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define RGBLIGHT_LIMIT_VAL 150 /* The maximum brightness level */
```

Comment:
```suggestion
#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_MOOD

```

Reply:
Hi, we want to keep #define RGBLIGHT_LAYERS
#define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF because we want to Capslock LED indicator on even we turn off the RGB mode.

Reply:
The defines have been moved to the `info.json` file with my previous suggestions.